<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PredictionDialog</class>
 <widget class="QDialog" name="PredictionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Venkat Machine Learning</string>
  </property>
  <widget class="QGraphicsView" name="imageView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Learn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>420</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>671</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Status: Idle</string>
   </property>
  </widget>
  <widget class="QLabel" name="actualLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Actual:</string>
   </property>
  </widget>
  <widget class="QLabel" name="predictedLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Predicted:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
